src/app/public/modules/ag-grid/types/cell-editor-autonumeric-params.ts

```diff
@@ -0,0 +1,6 @@
+import { SkyAutonumericOptions } from '@skyux/autonumeric';
+import { ICellEditorParams } from 'ag-grid-community';
+
+export interface SkyCellEditorAutonumericParams extends ICellEditorParams {
+  skyComponentProperties?: SkyAutonumericOptions;
```

Comment:
I wonder if we should nest the `SkyAutonumericOptions` inside of an object to get something like below.

I'm thinking of how we can support extra properties alongside direct autonumeric.js options like in the upcoming situation with [configurable debounce on the autonumeric directive
](https://github.com/blackbaud/skyux-autonumeric/pull/44).

```TS
export interface SkyCellEditorAutonumericParams extends ICellEditorParams {
  skyComponentProperties?: {
    autonumericOptions: SkyAutonumericOptions;
    // other properties
    debounceTime: number;
  }
}
```

Reply:
The Autocomplete follows the comment's pattern so maybe that is the right route to go.

![image](https://user-images.githubusercontent.com/10877317/112168608-d26c5580-8bc7-11eb-9e1f-065347e8a8df.png)